celery/app/base.py

```diff
@@ -733,8 +734,10 @@ def send_task(self, name, args=None, kwargs=None, countdown=None,
         options = router.route(
             options, route_name or name, args, kwargs, task_type)
         if expires is not None:
-            if isinstance(expires, datetime):
-                expires_s = (maybe_make_aware(expires) - self.now()).total_seconds()
+            if isinstance(expires, (str, datetime)):
```

Comment:
could you please update related tests for the proposed changes?

Reply:
@auvipy would that be just a new test in https://github.com/celery/celery/blob/26d7a4fa61f6ee36ad23cc3780e09a07eb350e8c/t/unit/tasks/test_tasks.py#L374 
Happy to send a PR once this is in

Reply:
you need to send pr along with it, i mean you can take over and come with a new pr including tests :) and tag me for initial review